pkg/execute/executor.go

```diff
@@ -234,19 +235,34 @@ func runPingCommand(args []string, clusterName string) string {
 	return fmt.Sprintf("pong from cluster '%s'", clusterName)
 }
 
-func showControllerConfig() (string, error) {
+func showControllerConfig() (configYaml string, err error) {
 	configPath := os.Getenv("CONFIG_PATH")
 	configFile := filepath.Join(configPath, config.ConfigFileName)
 	file, err := os.Open(configFile)
 	defer file.Close()
 	if err != nil {
-		return "", err
+		return configYaml, err
 	}
 
 	b, err := ioutil.ReadAll(file)
 	if err != nil {
-		return "", err
+		return configYaml, err
 	}
 
-	return string(b), nil
+	c := &config.Config{}
+	if len(b) != 0 {
+		yaml.Unmarshal(b, c)
+	}
+
+	// hide sensitive info
+	c.Communications.Slack.Token = ""
+	c.Communications.ElasticSearch.Password = ""
```

Comment:
if we have omitempty above why do we need to set empty strings again ?

Reply:
`omitempty` drops fields if they are empty while Marshaling. When we Unmarshal yaml to struct, we get Token and Password set in the struct. So in order to drop the fields, we need to make their value empty